rpm/SPECS/git-lfs.spec

```diff
@@ -6,7 +6,7 @@ Summary:        Git extension for versioning large files
 Group:          Applications/Archiving
 License:        MIT
 URL:            https://git-lfs.github.com/
-Source0:        https://github.com/github/git-lfs/archive/%{name}-%{version}.tar.gz
+Source0:        https://github.com/github/git-lfs/archive/v%{version}/%{name}-%{version}.tar.gz
```

Comment:
The URLs are actually `https://github.com/github/git-lfs/archive/{tag}.tar.gz`, like https://github.com/github/git-lfs/archive/v1.0.2.tar.gz. The "{name}-{version}" filename is set on a Content-Disposition header in the response.